tools/UpdatePaths/Scripts/68111_GAGs_clothesmate.txt

```diff
@@ -0,0 +1,11 @@
+#comment This replaces various clothing items with GAGs version https://github.com/tgstation/tgstation/pull/68111
+
+/obj/item/clothing/head/beanie/rasta : /obj/item/clothing/head/rasta
+/obj/item/clothing/head/beanie/waldo : /obj/item/clothing/head/waldo
+/obj/item/clothing/neck/stripedbluescarf : /obj/item/clothing/neck/large_scarf/blue
+/obj/item/clothing/neck/stripedgreenscarf : /obj/item/clothing/neck/large_scarf/green
+/obj/item/clothing/neck/stripedredscarf : /obj/item/clothing/neck/large_scarf/red
+/obj/item/clothing/head/beanie/stripedred : /obj/item/clothing/head/beanie/red
+/obj/item/clothing/under/pants/youngfolksjeans : /obj/item/clothing/under/pants/jeans
+/obj/item/clothing/under/pants/classicjeans : /obj/item/clothing/under/pants/jeans
+/obj/item/clothing/under/dress/blacktango : /obj/item/clothing/under/dress/tango
```

Comment:
```suggestion
/obj/item/clothing/head/beanie/rasta : /obj/item/clothing/head/rasta{@OLD}
/obj/item/clothing/head/beanie/waldo : /obj/item/clothing/head/waldo{@OLD}
/obj/item/clothing/neck/stripedbluescarf : /obj/item/clothing/neck/large_scarf/blue{@OLD}
/obj/item/clothing/neck/stripedgreenscarf : /obj/item/clothing/neck/large_scarf/green{@OLD}
/obj/item/clothing/neck/stripedredscarf : /obj/item/clothing/neck/large_scarf/red{@OLD}
/obj/item/clothing/head/beanie/stripedred : /obj/item/clothing/head/beanie/red{@OLD}
/obj/item/clothing/under/pants/youngfolksjeans : /obj/item/clothing/under/pants/jeans{@OLD}
/obj/item/clothing/under/pants/classicjeans : /obj/item/clothing/under/pants/jeans{@OLD}
/obj/item/clothing/under/dress/blacktango : /obj/item/clothing/under/dress/tango{@OLD}
```
You want to have `{@OLD}` on all of the paths you replace, otherwise UpdatePaths will discard things like pixel-shifting, name edits, or description edits in maps (AKA: years of soul will be removed if this script is run as-is and commited). Your manual update done earlier won't do this and that's fine, but this is important to ensure we aren't needlessly removing work. I just quickly ran through and added those all for you in this suggestion..

Reply:
i must have missed that in the updatepath guide thingy tyvm